VimeoUpload/Upload/Networking/Shared/VimeoRequestSerializer+SharedUpload.swift

```diff
@@ -93,7 +94,48 @@ extension VimeoRequestSerializer
         
         return [Constants.SizeKey: fileSizeString]
     }
+    
+    private func createMIMETypeParameters(url: URL) throws -> [String: Any]
+    {
+        let type: String
+        do
+        {
+            type = try url.MIMEType()
+        }
+        catch let error as NSError
+        {
+            throw error.error(byAddingDomain: UploadErrorDomain.Create.rawValue)
+        }
+        return [Constants.MIMETypeKey: type]
```

Comment:
Another nit pick, but we usually keep a new line above return, but you should do what's consistent in your codebase.